wilsonChan/assignments/treesBFS/lc637/lc637.py

```diff
@@ -0,0 +1,125 @@
+# Problem Statement #
+
+# Given a binary tree, populate an array to represent 
+# the averages of all of its levels.
```

Comment:
Great overview and examples. You could probably write one or two more test cases, especially for your edge cases.